ts/__init__.py

```diff
@@ -156,6 +160,10 @@ class PathHolders:
     'EndpointQualifier': [PathHolders.EndpointQualifier, PathHolders.EndpointQualifierCodec],
     'List_Long': [PathHolders.Long, PathHolders.ListLongCodec],
     'List_Integer': [PathHolders.ListIntegerCodec],
+    'ListCN_Boolean': [PathHolders.ListCNBooleanCodec],
```

Comment:
does nodejs implement sql yet? what is this change for?

Reply:
Removed.